### PR TITLE
GCP tests

### PR DESCRIPTION
**Pull request by trezorg**

### CODE REVIEW

File: tests/gcp/bundle_test.go [hunk 1]
```diff
@@ -1,6 +1,29 @@
 package gcp
 
-// Set PREFIX, GCP_PROJECT, and GOOGLE_APPLICATION_CREDENTIALS credentials before running these scripts
+/*
+Set PREFIX, GCP_PROJECT, and GOOGLE_APPLICATION_CREDENTIALS credentials before running these scripts
+Add or ensure next rpuoles are presents for google account member that is being used for terraform:
```
Comment (ArseniiPetrovich) [inline tests/gcp/bundle_test.go:5]: rpuoles => rules

**Review by ArseniiPetrovich — CHANGES_REQUESTED** [review]
I generally love the PR, but to make it merged we need to enable CI with these tests first. Please, check the following configuration: https://github.com/protofire/polkadot-failover-mechanism/blob/master/.circleci/config.yml , adjust it to include GCP tests, make sure all the tests are passed with the CI and then PR can be safely merged.
P.S. Contact me for any details needed 👍

File: tests/gcp/bundle_test.go (hunk 1 above)
Comment (trezorg) [reply tests/gcp/bundle_test.go]: Fixed

**Comment by trezorg** [comment]
> I generally love the PR, but to make it merged we need to enable CI with these tests first. Please, check the following configuration: https://github.com/protofire/polkadot-failover-mechanism/blob/master/.circleci/config.yml , adjust it to include GCP tests, make sure all the tests are passed with the CI and then PR can be safely merged.
> P.S. Contact me for any details needed

Hi.
https://app.circleci.com/pipelines/github/protofire/polkadot-failover-mechanism/141/workflows/8ec8e0f8-49a0-4286-8039-31cdfbf1e34a

Passed Circle CI tests for gcp-tests branch

**Review by ArseniiPetrovich — APPROVED** [review]
Awesome! LGTM!

**Pull request merged**